agent/config/types.go

```diff
@@ -29,10 +29,17 @@ type ImagePullBehaviorType int8
 type ContainerInstancePropagateTagsFromType int8
 
 type Config struct {
+
+	// PortRange Variable enables the clients to set their custom port range
+	// when there is no port range provided, this will default to the
+	// platform specific values
+	DynamicHostPortRange string
```

Comment:
please move this to the end of this struct, not the start. also update the comment